src/executePrettier.js

```diff
@@ -47,7 +47,7 @@ const executePrettierOnBufferRange = (editor: TextEditor, bufferRange: Range) =>
   const textToTransform = editor.getTextInBufferRange(bufferRange);
   const transformed = executePrettier(editor, textToTransform);
 
-  if (!transformed) return;
+  if (!transformed || transformed === textToTransform) return;
```

Comment:
Now that this conditional is a bit more complex, can we put this in a const to self-document (just for readability later on)?:

```js
const isTextUnchanged = transformed === textToTransform;
if (!transformed || isTextUnchanged) return;
```